docs/content/en/docs/yaml-crd-ref/app.md

```diff
@@ -3,3 +3,134 @@ title: KeptnApp
 description: Define all workloads and checks associated with an application
 weight: 10
 ---
+
+`KeptnApp` specifies a concept of a running application with a list of workloads and the possibility to define pre/post deployment and pre/post evaluations checks
+that should be executed at the Keptn application level.
+It contains information about all workloads and checks
+that are associated with a Keptn application.
+
+## Synopsis
+
+```yaml
+apiVersion: lifecycle.keptn.sh/v1alpha3
+kind: KeptnApp
+metadata:
+  name: <app-name>
+  namespace: <app-namespace>
+spec:
+  version: "x.y"
+  revision: x
+  workloads:
+  - name: <workload-name>
+    version: x.y.z
+  - name: podtato-head-left-leg
```

Comment:
we should either use placeholders or example names. Let's not combine these two styles together

Reply:
Fixed